src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java

```diff
@@ -347,46 +347,66 @@ public TesterDoesStepDefinitions(WebDriverWrapper driverWrapper) {
                 (String field, String value) ->
                         assertEquals(value, driverWrapper.getTextFromDefinitionList(field)));
 
+        And("^I check the registration number \\{([^\\}]+)\\} is shown within the Registration number span text",
```

Comment:
Should be changed to registration plate, not registration number - this is to adhear to business terminology